### ISSUE TITLE
force:source:push creates invalid package for Bot metadata

### ISSUE THREAD

**Issue by filiprafalowicz** [forcedotcom/cli#1487]
### Summary

When using new force:source:push command we are not able to deploy Bot metadata. Doing the same with force:source:legacy:push works as expected.

### Steps To Reproduce:

1. Create project with following files:
force-app/main/default/bots/My_Bot/My_Bot.bot-meta.xml
force-app/main/default/bots/My_Bot/v1.botVersion-meta.xml
1. Use new force:source:push command

### Expected result

Following package.xml file is generated:
```
<?xml version="1.0" encoding="UTF-8">
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
    <types>
        <members>My_Bot</members>
        <name>Bot</name>
    </types>
    <types>
        <members>My_Bot.v1</members>
        <name>BotVersion</name>
    </types>
    <version>54.0</version>
</Package>
```

And files included in the package look like this:
- package/bots/My_Bot.bot

### Actual result

Following package.xml file is generated:
```
<?xml version="1.0" encoding="UTF-8">
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
    <types>
        <members>My_Bot</members>
        <name>Bot</name>
    </types>
    <types>
        <members>My_Bot.v1</members>
        <members>v1</members>
        <name>BotVersion</name>
    </types>
    <version>54.0</version>
</Package>
```

And files included in the package look like this:
- package/bots/My_Bot.bot
- package/botVersions/v1.botVersion

### System Information

```json
{
        "cliVersion": "sfdx-cli/7.147.1",
        "architecture": "darwin-x64",
        "nodeVersion": "node-v16.14.2",
        "pluginVersions": [
                "@oclif/plugin-autocomplete 0.3.0 (core)",
                "@oclif/plugin-commands 1.3.0 (core)",
                "@oclif/plugin-help 3.3.1 (core)",
                "@oclif/plugin-not-found 1.2.6 (core)",
                "@oclif/plugin-plugins 1.10.11 (core)",
                "@oclif/plugin-update 1.5.0 (core)",
                "@oclif/plugin-warn-if-update-available 1.7.3 (core)",
                "@oclif/plugin-which 1.0.4 (core)",
                "@salesforce/lwc-dev-server 2.11.0",
                "@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)",
                "alias 2.0.0 (core)",
                "apex 0.11.0 (core)",
                "auth 2.0.1 (core)",
                "community 1.1.4 (core)",
                "config 1.3.30 (core)",
                "custom-metadata 1.0.12 (core)",
                "data 0.6.13 (core)",
                "generator 1.2.2 (core)",
                "info 2.0.0 (core)",
                "limits 2.0.0 (core)",
                "org 1.11.2 (core)",
                "salesforce-alm 54.2.0 (core)",
                "schema 2.0.0 (core)",
                "sfdx-cli 7.147.1 (core)",
                "signups 1.0.0 (core)",
                "source 1.9.3 (core)",
                "telemetry 1.4.0 (core)",
                "templates 54.3.0 (core)",
                "trust 1.1.0 (core)",
                "user 1.7.1 (core)"
        ],
        "osVersion": "Darwin 21.3.0"
}
```

### Additional information

Push will fail with `An object 'v1' of type BotVersion was named in package.xml, but was not found in zipped directory` error message.

**Comment by aswin-accellor** [comment]
This issue is also happening for force:source:deploy command

**Comment by iowillhoit** [comment]
Hello @filiprafalowicz and @aswin-popcornapps, this issue should be resolved in yesterday's latest-rc (`7.150.0`)
Please give it a try and let us know if are still seeing issues. Thanks!

**Comment by iowillhoit** [comment]
CLI version `7.150.0` has been promoted to latest. Closing this issue.
